docs/pom.xml

```diff
@@ -105,6 +108,22 @@
                     <siteArchiveSkip>${helidon.sitegen.skip}</siteArchiveSkip>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Is this used ?

Reply:
Not if the build does not generated the projects. I've removed it.